README.md

```diff
@@ -1,2 +1,25 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group consists of corporate representatives in leadership roles responsible for managing for large-scale deployments of Node.js in an Enterprise environment. We strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the needs of Node.js Enterprise users.
+
+Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month during the formation process.
+
+### Governance and Contributing
+The Enterprise Advisory Group has adopted the core governance and contributing policies of Node.js. This group is a part of the [Node.js End User Feedback Initiative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee).
+
+Members are expected to provide feedback on how proposed changes in the Node.js project will affect Enterprise teams, business goals, and application performance at scale. Members attend regular group meetings to provide feedback and respond to occasional outreach requests. 
+
+### Enterprise Advisory Group Team Members
+
+* [dshaw](https://github.com/dshaw) - **Dan Shaw** &lt;dshaw@dshaw.com&gt; - Node.js End User Feedback coordinator - Node.js at Large ✨
+* [jasnell](https://github.com/jasnell) - **James Snell** - TSC - nearForm
```

Comment:
@dshaw ... I shouldn't be listed as a member here :-)

Reply:
Please list Barry Woods (https://github.com/woodsbarry) as the participant from nearForm :-)